wolfssl/wolfcrypt/asn.h

```diff
@@ -2886,9 +2883,13 @@ struct DecodedCRL {
     int          version;            /* version of cert    */
     void*   heap;
 #ifndef NO_SKID
-    byte    extAuthKeyIdSet;
     byte    extAuthKeyId[SIGNER_DIGEST_SIZE]; /* Authority Key ID        */
+    byte    extAuthKeyIdSet:1;
 #endif
+    byte    crlNumberSet:1;          /* CRL number set indicator */
+    byte    lastDateFormat:1;        /* format of last date */
```

Comment:
Is `lastDateFormat` and `nextDateFormat` actually a bit?

Reply:
No they're not, changed back